### DISCUSSION TITLE
Is it possible to configure SAML app without metadata url/file?

### DISCUSSION THREAD

**Discussion by pschiffe** [zitadel/zitadel#6427] [Q&A]
Hello, I'm trying to log in to Wazuh via Zitadel, but it seems that they don't provide any SAML metadata xml file. Their docs contain only direct configuration of saml parameter, for example here for Keycloack:
https://documentation.wazuh.com/current/user-manual/user-administration/single-sign-on/administrator/keycloak.html

Do you have any advice how to proceed in this case? Cheers.

**Comment by hifabienne** [comment]
Hei @pschiffe 
At the moment it is not possible to add a SAML app without the metadata file.
ZITADEL needs some information that it works.
You can create a file yourself and fill the information needed and upload afterwards.
You can find a minimal configuration here: https://zitadel.com/docs/guides/integrate/services/pingidentity-saml#zitadel-create-the-application

**Comment by harshavmb** [comment]
I got this working with action & flows, documented at [Wazuh SSO with Zitadel using SAML 2.0
](https://harshanu.space/en/tech/wazuh-zitadel-sso/). I see you opened another issue at Wazuh [here](https://github.com/wazuh/wazuh/issues/18630).
